docs/source/monitoring_stack.rst

```diff
@@ -246,6 +246,29 @@ will use the user permissions that runs it. This is important if you want to pla
 
 In that case, you need to create the directory before calling ``start-all.sh`` and make sure it has the right permissions for the user running the command.
 
+Add Additional Prometheus Targets
+....................................
+There are situations where you would like to monitor additional targets using the Prometheus server of the monitoring stack.
+The Prometheus server reads its targets from file, this file is generated from a template when calling ``start-all.sh``. 
```

Comment:
```suggestion
The Prometheus server reads its targets from a file, this file is generated from a template when calling ``start-all.sh``. 
```

Reply:
when is the start-all.sh file normally called?